examples/demo_threading/client.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
```

Comment:
The behavior of this client script is very different from what I expected.
I would expect this to spin up a single client that runs for as-long as the user wants.

That would make it easy to spin up as many clients as needed, pause them, kill them, observe their behavior, etc.
We could also have a "nastyclient.py" that tries to inject false events, scrambled json, etc.